app.js

```diff
@@ -2,108 +2,99 @@
 if (process.env.NODE_ENV !=='production'){ // if the environment is development include env file
     require('dotenv').config(); // this will load all environment processes
 }
-
-// Dependencies
+/**
+ * -----------------DEPENDENCIES-------------------
+ */
 const express = require('express');
 const path = require('path');
-const passport = require('passport');
-const bodyParser = require('body-parser');
-const LocalStrategy = require('passport-local');
+const mysql = require('mysql');
 const bcrypt = require('bcrypt');
-const session = require('express-session');
+const bodyParser = require('body-parser');
```

Comment:
I think body-parser is deprecated on express.js
https://medium.com/@mmajdanski/express-body-parser-and-why-may-not-need-it-335803cd048c

Reply:
gotcha ill go ahead and fix that